ros2_batch_job/__main__.py

```diff
@@ -408,9 +417,11 @@ def run(args, build_function, blacklisted_package_names=None):
 
     args.white_space_in = args.white_space_in or []
     args.workspace = 'work space' if 'workspace' in args.white_space_in else 'ws'
-    args.sourcespace = 'source space' if 'sourcespace' in args.white_space_in else 'src'
-    args.buildspace = 'build space' if 'buildspace' in args.white_space_in else 'build'
-    args.installspace = 'install space' if 'installspace' in args.white_space_in else 'install'
+    for dir_name in [args.source_space, args.build_space, args.install_space]:
+        dir_name.rstrip('/' + os.sep)
```

Comment:
`rstrip` isn't an in-place change - it returns the modified value.

Reply:
Gah. Added a nested function instead